### DISCUSSION TITLE
Welcome to Koalagator Discussions

### DISCUSSION THREAD

**Discussion by evolve2k** [koalagator/koalagator#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jharvey13** [comment]
As a person trying to roll out a community calendar in a rural American county—where FB serves as the main form of advertising events—I'm very much looking forward to koalagator. I'm not skilled enough to contribute (haven't seriously written any code in 15+ years) but will be happy to deploy versions, test, and provide real world feedback.

**Reply by evolve2k** [reply]
Thanks for your support. Yes we're still getting things setup but appreciate the encouragment. An event calendar for a rural town sounds like a good use case, any more details you want to share on why koalagator seems attractive for your needs? Helps us to better understand how people want to use this tool.

**Comment by jharvey13** [comment]
The main draw, aside from it being RoR, is it does one thing. No comments. No likes. No pretense it’s building a community. It’s a calendar, in chronological order, and that’s it. There’s no algorithm trying to figure out what events you may or may not want to attend.In my ideal world, the only other features I’d enjoy would be moderation controls, i.e. letting a random user submit an event, but hold it for review. The ability to add an graphic to events—many around here have a GIF or PNG flier. And, in a perfect world, a weekly “Here’s What’s Happening” email to users who’ve opted in, with a simple list of what’s happening in the next seven days.On Jun 5, 2024, at 11:29 PM, Richie Khoo ***@***.***> wrote:﻿
Thanks for your support. Yes we're still getting things setup but appreciate the encouragment. An event calendar for a rural town sounds like a good use case, any more details you want to share on why koalagator seems attractive for your needs? Helps us to better understand how people want to use this tool.
